use_case_examples/deployment/cifar_8_bit/Makefile

```diff
@@ -1,17 +0,0 @@
-# Useful for jupyter notebooks
```

Comment:
same here. I think this deploys the "cifar_from_scratch" one

Reply:
I deleted this one. cifar 8 bit is gone

Reply:
but it was changed to deploy cifar from scratch actually

Reply:
Got it. I changed the naming. Looks like this one needs a good refresh to run.